src/main/java/com/specialwarriors/conal/common/auth/service/AuthService.java

```diff
@@ -0,0 +1,50 @@
+package com.specialwarriors.conal.common.auth.service;
+
+import com.specialwarriors.conal.common.LogService;
+import com.specialwarriors.conal.common.auth.exception.AuthException;
+import com.specialwarriors.conal.common.auth.exception.CustomAuthException;
+import com.specialwarriors.conal.common.auth.jwt.JwtProvider;
+import com.specialwarriors.conal.common.auth.jwt.JwtTokenResponse;
+import com.specialwarriors.conal.common.auth.jwt.domain.RefreshToken;
+import com.specialwarriors.conal.common.auth.jwt.repository.RefreshTokenRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class AuthService {
+
+    private final JwtProvider jwtProvider;
+    private final RefreshTokenRepository refreshTokenRepository;
+    private final LogService logService;
+
+    public JwtTokenResponse reissueToken(String refreshToken) {
+        Long userId = jwtProvider.getUserId(refreshToken);
+
+        RefreshToken savedToken = refreshTokenRepository.findRefreshTokenByUserId(userId)
+                .orElseThrow(() -> new CustomAuthException(AuthException.INVALID_TOKEN));
+
+        if (!savedToken.getRefreshToken().equals(refreshToken)) {
+            throw new CustomAuthException(AuthException.INVALID_TOKEN);
+        }
+
+        boolean isRefreshExpired = jwtProvider.isExpired(refreshToken);
+
+        if (isRefreshExpired) {
+            String newAccessToken = jwtProvider.createAccessToken(userId);
+            String newRefreshToken = jwtProvider.createRefreshToken(userId);
+
+            savedToken.updateRefreshToken(newRefreshToken);
+
+            log.info("[{}}] accessToken과 refreshToken을 재발행했습니다", logService.getClassName());
```

Comment:
```suggestion
            log.info("[{}}] accessToken과 refreshToken을 재발행했습니다", this.getClassName());
```
로그를 남기는 의도가 기록하는 작업이 발생하는 클래스 위치를 명시하려는 것이라면 `AuthService`를 기록하는 것이 맞는 것 같습니다. 제가 오해했을 수도 있으니 검토하고 의견 부탁드립니다.